test/test_source_comment_associator.rb

```diff
@@ -1,3 +1,9 @@
+#! /usr/bin/env ruby
+# coding: utf-8
+# frozen_string_literal: true
+# warn_indent: true
```

Comment:
`warn_ident`? `warn_past_scope`?

Reply:
Ah, I missed these unexpected changes - seems Pragmater got confused. I'll fix that up.